### ISSUE TITLE
NTR: Summer (hot weather) onset (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7836]
New term request Summer (hot weather) onset (samant_101217013718): Warmer; lessening in cold weather  UMLS: 2039366

**Comment by pnrobinson** [comment]
@MickeySegal
from the term request, it seems that this might be referring to Exacerbated by hot weather (i.e., repeatedly, rather than the initial onset of a disease or feature being in Summer)?

**Comment by MickeySegal** [comment]
No, this is for infections that spread in summer, e.g., West Nile, RMSF, and various encaephaldites.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding as
label: History of disease onset in summer
id: HP:0020009
def: The initial manifestation of a disease or symptom occurring during the summer season.
